Reference/Scheduling/index-v11.md

```diff
@@ -0,0 +1,180 @@
+:::
+
+## RecurringHostedService example
+
+```C#
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Logging;
+using Umbraco.Cms.Core.Scoping;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Core.Sync;
+using Umbraco.Cms.Infrastructure.HostedServices;
+
+namespace Umbraco.Docs.Samples.Web.RecurringHostedService;
+
+public class CleanUpYourRoom : RecurringHostedServiceBase
+{
+    private readonly IRuntimeState _runtimeState;
+    private readonly IContentService _contentService;
+    private readonly IServerRoleAccessor _serverRoleAccessor;
+    private readonly IProfilingLogger _profilingLogger;
+    private readonly ILogger<CleanUpYourRoom> _logger;
+    private readonly ICoreScopeProvider _scopeProvider;
+
+    private static TimeSpan HowOftenWeRepeat => TimeSpan.FromMinutes(5);
+    private static TimeSpan DelayBeforeWeStart => TimeSpan.FromMinutes(1);
+
+    public CleanUpYourRoom(
+        IRuntimeState runtimeState,
+        IContentService contentService,
+        IServerRoleAccessor serverRoleAccessor,
+        IProfilingLogger profilingLogger,
+        ILogger<CleanUpYourRoom> logger,
+        ICoreScopeProvider scopeProvider)
+        : base(logger, HowOftenWeRepeat, DelayBeforeWeStart)
+    {
+        _runtimeState = runtimeState;
+        _contentService = contentService;
+        _serverRoleAccessor = serverRoleAccessor;
+        _profilingLogger = profilingLogger;
+        _logger = logger;
+        _scopeProvider = scopeProvider;
+    }
+
+    public override Task PerformExecuteAsync(object? state)
+    {
+        // Don't do anything if the site is not running.
+        if (_runtimeState.Level is not RuntimeLevel.Run)
+        {
+            return Task.CompletedTask;
+        }
+
+        // Wrap the three content service calls in a scope to do it all in one transaction.
+        using ICoreScope scope = _scopeProvider.CreateCoreScope();
+            
+        int numberOfThingsInBin = _contentService.CountChildren(Constants.System.RecycleBinContent);
+        _logger.LogInformation("Go clean your room - {ServerRole}", _serverRoleAccessor.CurrentServerRole);
+        _logger.LogInformation("You have {NumberOfThingsInTheBin} items to clean", numberOfThingsInBin);
+
+        if (_contentService.RecycleBinSmells())
+        {
+            // Take out the trash
+            using (_profilingLogger.TraceDuration<CleanUpYourRoom>("Mum, I am emptying out the bin",
+                       "It's all clean now"))
+            {
+                _contentService.EmptyRecycleBin(userId: -1);
+            }
+        }
+
+        // Remember to complete the scope when done.
+        scope.Complete();
+        return Task.CompletedTask;
+    }
+}
+```
+:::note
+If you are using an Umbraco version before v9.4 you can't pass in an instance of `ILogger` in to the base constructor. See the code example below:
+
+```C#
+public CleanUpYourRoom(
+    ...)
+    : base(HowOftenWeRepeat, DelayBeforeWeStart)
+{
+    ...
+}
+```
+:::
```

Comment:
```suggestion
```

We might want to remove this since this doc is for v11 specifically. 

Also this example looks a bit odd [on the live site](https://our.umbraco.com/documentation/Reference/Scheduling/#recurringhostedservice-example) even though the markdown is correct

Reply:
I agree that this should be removed from the v11 version of the article 💪 
Have no idea why the formatting of this code sample is funky on the frontend - but I've seen it before, and it seems related to the fact that the code sample is inside a note 🤷‍♀️ 😅